src/modules/charging-import/lib/queries/charging.js

```diff
@@ -132,7 +132,71 @@ on conflict (external_id) do update set
   date_updated=NOW();
 `;
 
+const importChargeVersions = `INSERT INTO water.charge_versions (licence_ref, scheme, external_id, version_number, start_date, status, apportionment,
+error, end_date, billed_upto_date, region_code, date_created, date_updated, source, invoice_account_id, company_id, licence_id, change_reason_id)
+SELECT
+  l."LIC_NO" AS licence_ref,
+  'alcs' AS scheme,
+  concat_ws(':', v."FGAC_REGION_CODE", v."AABL_ID", v."VERS_NO") as external_id,
```

Comment:
I think this should be `cvm.external_id` as for the gaps, this won't be populated otherwise